src/openPr.js

```diff
@@ -24,7 +25,21 @@ const addArtifact = async (inputs, releaseId) => {
   return artifact
 }
 
-const createDraftRelease = async (inputs, newVersion) => {
+const getReleaseNotes = async (inputs, newVersion) => {
```

Comment:
as a consequence of generateReleaseNotes not needing the whole inputs, I believe this function also won't need the whole inputs object